interpreter/namerd/src/main/scala/io/buoyant/namerd/iface/NamerdInterpreterInitializer.scala

```diff
@@ -83,9 +110,10 @@ case class NamerdInterpreterConfig(
     val stats = stats0.scope(label)
 
     val tlsParams = tls.map(_.params).getOrElse(Stack.Params.empty)
+    val failureThresholdParams = failureThreshold.map(_.params).getOrElse(StackParams.empty)
```

Comment:
Let's fall back to our defaults if the failureTreshold is absent, instead of omitting the param entirely and letting Finagle pick the default.